unifiedpush_android/lib/unifiedpush_android.dart

```diff
@@ -0,0 +1,133 @@
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:unifiedpush_platform_interface/unifiedpush_platform_interface.dart';
+
+import 'CallbackDispatcher.dart';
+import 'constants.dart';
+
+
+class UnifiedPushAndroid extends UnifiedPushPlatform {
+  static void registerWith() {
+    UnifiedPushPlatform.instance = UnifiedPushAndroid();
+  }
+  
+  static SharedPreferences? _prefs;
+  static get prefs async {
+    if (_prefs == null) {
+      _prefs = await SharedPreferences.getInstance();
+    }
+    return _prefs;
+  }
+
+  static const MethodChannel _channel = MethodChannel(PLUGIN_CHANNEL);
+
+  static void Function(String token, String endpoint)? _onNewEndpoint = (String _, String e) {};
+  static void Function(String token, String? message)? _onRegistrationRefused = (String _, String? r) {};
+  static void Function(String token, String? message)? _onRegistrationFailed = (String _, String? r) {};
+  static void Function(String token)? _onUnregistered = (String _) {};
+  static void Function(String token, String message)? _onMessage = (String _, String m) {};
+
+  @override
+  Future<List<String>> getDistributors() async {
+    return (await _channel.invokeMethod(PLUGIN_EVENT_GET_DISTRIBUTORS))
+        .cast<String>();
+  }
+
+  @override
+  Future<void> registerApp(String distributor, String token) async {
+    await _channel.invokeMethod(PLUGIN_EVENT_REGISTER_APP, [distributor, token]);
+  }
+  
+  @override
+  Future<void> unregister(String token) async {
+    await _channel.invokeMethod(PLUGIN_EVENT_UNREGISTER, [token]);
```

Comment:
It seems like there's a mismatch between this which sends one token, and the android part which expects two things
https://github.com/UnifiedPush/flutter-connector/pull/56/files#diff-063b58b3b219380864d6ba49557ac9e74ff71fe5a151163a7f229c4462311d33R70

Reply:
Good catch thanks.